explorer/src/providers/accounts/block.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
```

Comment:
Can you please move this file from `./providers/accounts/block.tsx` to `./providers/blocks.tsx`?